stdlib/asyncio/mixins.pyi

```diff
@@ -0,0 +1,8 @@
+import threading
+from typing import Any
+
+_global_lock: threading.Lock
+_marker: object
+
+class _LoopBoundMixin:
+    def __init__(self, *, loop: Any = ...) -> None: ...
```

Comment:
Maybe we should just skip the `loop` argument, It can't really be used. Then we can also remove `_marker` above.